akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -9,6 +9,7 @@ import scala.util.control.NonFatal
 import akka.stream.stage._
 import akka.stream.Supervision
 import akka.stream.impl.ReactiveStreamsCompliance
+import akka.stream.FlowMaterializer
```

Comment:
The extra parameter would have to be on all Stages, which is a no-go: this is an implementation detail that has no business in the user API. The alternative is to provide the “current” FlowMaterializer in a ThreadLocal, but that is too high a price in terms of ugliness.